packages/cactus-plugin-satp-hermes/src/main/typescript/blo/admin/get-healthcheck-handler-service.ts

```diff
@@ -0,0 +1,41 @@
+import { GetStatusError } from "../../core/errors/satp-errors";
+import { HealthCheckResponse } from "../../generated/gateway-client/typescript-axios/api";
+import { Logger } from "@hyperledger/cactus-common";
+import { SATPManager } from "../../gol/satp-manager";
+
+export async function ExecuteGetHealthCheck(
+  logger: Logger,
```

Comment:
@AndreAugusto11 I recommend passing in the log level instead and creating your own logger with a label that matches the function name so that in the logs it's easier to find later on.

An example of how I usually do it can be seen here:
`packages/cactus-plugin-ledger-connector-besu/src/main/typescript/impl/get-block-v1/get-block-v1-impl.ts`

https://github.com/hyperledger-cacti/cacti/blob/ff842d2a59ef898e734e1424ee6f26b52ba0af9b/packages/cactus-plugin-ledger-connector-besu/src/main/typescript/impl/get-block-v1/get-block-v1-impl.ts

Applies throughout the PR/codebase as a suggestion.